include/roboteq_controller.hpp

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include <memory> 
+
+#include "i_comm.hpp"
+
+namespace roboteq {
+
+    class roboteq_controller {
+
+        public :
+        
+        explicit roboteq_controller(std::unique_ptr<i_comm> &&comm);
+        ~roboteq_controller(){}
```

Comment:
should explicitly delete the copy constructor because the class has a unique ptr member var and unique ptrs cant be copy constructed.

`roboteq_controller ( const roboteq_controller & ) = delete;`

copy assignment is also invalid because lvalue copying of a unique ptr is invalid so the copy assignment operator should be deleted.

`roboteq_controller & operator= ( const roboteq_controller& ) = delete;`

Reply:
![image](https://user-images.githubusercontent.com/56097508/69999687-38dae980-1527-11ea-9429-bddd4e79c0a5.png)

Reply:
you still need to delete the copy constructor
`roboteq_controller ( const roboteq_controller & ) = delete;`

Reply:
bump.

copy constructor deletion still missing.
roboteq_controller ( const roboteq_controller & ) = delete;

Reply:
I add this in the lastest commit (Jan_08)